interface-definitions/pki.xml.in

```diff
@@ -35,6 +35,12 @@
               <multi/>
             </properties>
           </leafNode>
+          <leafNode name="system-install">
```

Comment:
Is there a reason to make this explicit and not implicit?

Reply:
Might not always be desirable.

E.g. installing CA for third-party VPN, you likely don't want it trusted for unrelated https traffic.

Reply:
Good example!